filters-share/specific_ELEMHIDE.txt

```diff
@@ -59,6 +59,9 @@ sports.donga.com##div[id*="_AD"]
 ! ##div[id$="_Ad"]
 yes24.com##div[id$="_Ad"]
 !
+hogaeng.co.kr##.bo_notice:has(.spon_b_icon1)
```

Comment:
Currently, Mozilla Firefox does not support `:has()` widely.

Reply:
so how to do?

Reply:
Please move to `/filters-share/extended_css_ELEMHIDE.txt`.

Reply:
sry its my mistake